### PR TITLE
MSTR-38: Show empty status in App Selector

### PR DESCRIPTION
**Pull request by guillegr123**

### CODE REVIEW

File: src/apps/common/submodules/appSelector/appSelector.scss [hunk 1]
```diff
@@ -179,38 +183,82 @@
...
+					&.app-item {
+						background-color: $panel-background-color;
+						cursor: pointer;
+						display: flex;
+						flex-wrap: wrap;
+						box-shadow: inset 0px 1px 0px 0px $white,
+							inset 0px -1px 0px 0px $white;
+
+						&:hover, &.selected {
+							background-color: $french-pass;
```
Comment (azefiel) [inline src/apps/common/submodules/appSelector/appSelector.scss]: Check the mock up again, the `:hover` color a slightly lighter the `.selected` one (use `lighten()`).

File: src/apps/common/submodules/appSelector/appSelector.scss [hunk 2]
```diff
@@ -179,38 +183,82 @@
...
+							margin-right: 0.75rem;
+							margin-bottom: 0.75rem;
+						}
+						.app-title {
+							flex: 1;
+							margin-bottom: 0.75rem;
+							align-self: center;
+						}
+						.app-description {
+							width: 100%;
```
Comment (azefiel) [inline src/apps/common/submodules/appSelector/appSelector.scss]: App descriptions are actually truncated to only appear on a single line. If I had to guess, I'd say the `max-width` is about 75%.

To achieve the ellipsis effect, you can combine the following properties:
```suggestion
							width: 75%;
							white-space: nowrap;
							overflow-x: hidden;
							text-overflow: ellipsis;
```

Comment (guillegr123) [reply src/apps/common/submodules/appSelector/appSelector.scss]: Thanks, I'll truncate the descriptions this way.

File: src/apps/common/submodules/appSelector/appSelector.scss (hunk 1 above)
Comment (guillegr123) [reply src/apps/common/submodules/appSelector/appSelector.scss]: I've missed it, I'll change the `:hover` color.

File: src/apps/common/submodules/appSelector/appSelector.scss [hunk 3]
```diff
@@ -206,6 +206,10 @@
 
 						&.selected {
 							background-color: $french-pass;
+
+							&:hover {
+								background-color: darken($french-pass, 5%);
```
Comment (azefiel) [inline src/apps/common/submodules/appSelector/appSelector.scss]: Mmmm, actually I don't think the color is different when hovering a selected item.

Comment (guillegr123) [reply src/apps/common/submodules/appSelector/appSelector.scss]: Yes, it was not in the mock-up either. I'll remove it. It was just that it felt a little awkward to have a color change when hovering the other items, but not when hovering the selected ones.

**Pull request merged**